cookbooks/van-keken-vof/doc/van-keken-vof.tex

```diff
@@ -0,0 +1,215 @@
+    Figure~\ref{fig:vk-2}. However, the above results also compare extremely favorably with the 
+    computations with smoothed, continuous initial conditions for the compositional field on 
+    the right in Figure~\ref{fig:vk-2}.
+    As in Figure~\ref{fig:vk-2}, 5 global refinements correspond to a $32 \times 32$ mesh and 10 global refinements correspond to a $1024 \times 1024$ mesh.}
+    \label{fig:vof-vk-1}
+\end{figure}
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.4\textwidth]{cookbooks/van-keken-vof/doc/vof_van_keken_refinement_comparison.png}
+    \caption{\it The results of two computations of the van Keken problem made with the VOF 
+        interface tracking algorithm overlaid upon each other at $t_\text{end}=2000$.
+        This visualization shows the reconstructed boundary between the two
+        materials at the final time $t_\text{end}$ as computed on a uniform grid
+        with 7 and 8 levels of refinement.
+        The boundaries between the materials are displayed as contours
+        of the fields $\tilde{\psi}^7(t_\text{end})$ (black) and
+        $\tilde{\psi}^8\,(t_\text{end})$ (bright green), which are generated by the  visualization postprocessor.
+        The contours for the reconstructed material boundaries are superimposed
+        on a color gradient visualization of the material composition for the
+        computation with 8 levels of refinement in order to make the regions
+        with each fluid type more evident.
+        Compare with the fourth image on the right in Figure~\ref{fig:vk-1}.
+    }
+    \label{fig:VOF_van_Keken-02}
+\end{figure}
+
+As noted above, when the interface is discontinuous, the van Keken problem is a version 
+of the Rayleigh-Taylor problem, which is unstable to perturbations of all 
+wavelengths\footnote{This is true whether the two fluids have the same viscosity or different 
+viscosities.} (e.g. see~\cite{SC:1961}).
+Therefore, it is extremely sensitive to the initial conditions.
+In order to address this sensitivity, we do not use  the default approach of
+computing the initial material volume fractions using a composition quadrature.
+Instead we compute the initial volume fractions using a signed distance
+function $\phi$ as follows~\cite{JMR:2019,JMR-EGP:2019}.
+
+First we create the function $\phi$, which has the following two properties: 1) it is positive 
+in the region that contains one of the fluids, which we will refer to as fluid~1, and negative 
+in the complement of this region, which we will refer to as fluid~2, and 2) at each point in 
+the domain the magnitude $| \phi |$ of $\phi$ is the distance to the boundary between the 
+two fluids or materials.
+In the computations shown here, we use an approximation $\tilde{\phi}$ to $\phi$ such that   
+difference between $\tilde{\phi}$ and $\phi$ is small enough for the purposes of making the 
+computations high-quality.
```

Comment:
```suggestion
computations high-quality.
The use of an approximation as opposed to the function itself is due to obtaining an appropriate function for almost all nontrivial (not a line or a circle) boundaries is extremely difficult.
In this case, because the boundary is close to horizontal, we use the vertical distance to the boundary based on the argument that the gradient will not differ sufficiently from $1$ to induce errors in the initialization computation.
```
Attempted elaboration to clari

Reply:
@class4kayaker The gradient is a vector, not a scalar.  I think that you meant 

> ... the gradient of $\phi$ will not differ sufficiently from $(1,0)$ so as to introduce large errors in the initialization computation.